README.md

```diff
@@ -134,6 +134,14 @@ Download the latest binary executable for your operating system:
   dpkg -i tektoncd-cli-0.16.0_Linux-64bit.deb
   ```
 
+### NixOS/Nix
+
+You can install `tektoncd-cli` from [nixpkgs](https://github.com/NixOS/nixpkgs) on any system that supports the `nix` package manager.
```

Comment:
Oh waw it got in 😅 I kept updating my own derivation ([here](https://git.sr.ht/~vdemeester/home/tree/master/item/nix/packages/tkn/default.nix)), I guess I should update nixpkgs from now on 😛 (I wash shy to propose it at first, and then time did bite me)

Reply:
You're welcome to add yourself as a maintainer :)